docs-source/content/userguide/managing-domains/domain-in-image/base-images/_index.md

```diff
@@ -88,38 +88,51 @@ Oracle Container Registry.
-This patch does have some prerequisite patches that will also need to be applied. The standard image `container-registry.oracle.com/middleware/weblogic:12.2.1.3` already has all of these patches applied.
+The Oracle WebLogic Server Kubernetes Operator and WebLogic Server 12.2.1.3 image require patch 29135930.
+This patch has some prerequisite patches that will also need to be applied. To create and customize the WebLogic Server image,
+and apply the required patches, use the [WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool).
 
-[This sample](https://github.com/oracle/docker-images/blob/master/OracleWebLogic/samples/12213-patch-wls-for-k8s/README.md) in
-the Oracle GitHub Docker images repository demonstrates how to create an image with arbitrary patches, starting from an unpatched WebLogic Server 12.2.1.3 image (not the standard `container-registry/middleware/weblogic:12.2.1.3` pre-patched image).  You can customize that sample to apply a different set of patches, if you require additional patches or PSUs.
+To use the Image Tool, follow the [Setup](https://github.com/oracle/weblogic-image-tool/blob/master/README.md#setup) instructions
+and [Quick Start](https://github.com/oracle/weblogic-image-tool/blob/master/site/quickstart.md) Guide.
 
-When using that sample, you will need to download the required patch and also
-some prerequisite patches.  To find the correct version of the patch, you should
-use the "Product or Family (Advanced)" option, then choose "Oracle WebLogic Server"
-as the product, and set the release to "Oracle WebLogic Server 12.2.1.3.181016" as
-shown in the image below:
+To build the WebLogic Server image and apply the patches:
 
-![patch download page](/weblogic-kubernetes-operator/images/patch-download.png)
+1. Add the Server JRE and the WebLogic Server installer to the [Cache Tool](https://github.com/oracle/weblogic-image-tool/blob/master/site/cache.md).
 
+    ```
+    $imagetool cache addInstaller \
+    --type jdk \
+    --version 8u202 \
+    --path /home/acmeuser/wls-installers/jdk-8u202-linux-x64.tar.gz
 
-The `Dockerfile` in that sample lists the base image as follows:
+    $imagetool cache addInstaller \
+    --type wls \
+    --version 12.2.1.3.0 \
+    --path /home/acmeuser/wls-installers/fmw_12.2.1.3.0_wls_Disk1_1of1.zip
+    ```
+2. Use the [Create Tool](https://github.com/oracle/weblogic-image-tool/blob/master/site/create-image.md)
+to build the image and apply the patches.
 
-```
-FROM oracle/weblogic:12.2.1.3-developer
-```
+    For the Create Tool to download the patches, you must pass in your My Oracle Support credentials.
+    ```
 
-You can change this to use the standard WebLogic Server image you
-downloaded from the OCR by updating the `FROM` statement
-as follows:
+    $imagetool create \
+    --tag weblogic:12.2.1.3 \
+    --type=wls \
+    --version 12.2.1.3.0  \
+    --jdkVersion=8u241 \
+    -–patches=29135930_12.2.1.3.0,27117282_12.2.1.3.0 \
+    --user  username.mycompany.com \
```

Comment:
remove extra space

Reply:
Is the extra space after --version 12.2.1.3.0, the same extra space as noted in the previous comment? (Thanks Mr. Eagle Eyes!)

Reply:
there is one after `--user` too

Reply:
Yep, thanks!